components/script/script_thread.rs

```diff
@@ -1195,7 +1195,7 @@ impl ScriptThread {
 
         if let Some(fragment) = doc.url().fragment() {
             self.check_and_scroll_fragment(fragment, pipeline, doc);
-        }
+        };
```

Comment:
This is to appease borrow checker on `RefCell` in `if let`. See https://github.com/rust-lang/rust/issues/22449